confluent_kafka/schema_registry/avro.py

```diff
@@ -0,0 +1,227 @@
+import json
+import struct
+from copy import deepcopy
+
+from fastavro import parse_schema, schemaless_writer, schemaless_reader
```

Comment:
in .net i have separate packages for avro, protobuf and json (also schema registry) because e.g. people using protobuf won't want dependencies on avro. not sure how much of a concern it is in python, but seems important enough to me.